packages/azblob-async/azblob-async.0.1.0/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+synopsis: "A trivial Azure Blob Storage interface for OCaml"
+maintainer: "obiwanko@me.com"
+authors: "Kazuo Koga"
+license: "ISC"
+homepage: "https://github.com/kkazuo/azblob"
+bug-reports: "https://github.com/kkazuo/azblob/issues"
+depends: [
+  "ocaml" {>= "4.06.0"}
```

Comment:
Is it because of https://github.com/ocaml/opam-repository/pull/15407 that you added this constraint here?

Reply:
Yes. It related.
I've read this discussion: https://discuss.ocaml.org/t/ann-dune-2-0-0/4758/4
In my understanding, I will choice 'dune < 2.0' or 'ocaml >= 4.06' for now.
I've choiced latter.
Any advice? I'm glad to hear anything.

Reply:
This PR (https://github.com/ocaml/opam-repository/pull/15410) adds a way to install dune 2.0.0 on 4.05 and older. I'm going to merge your PR but in case you want to be compatible with OCaml < 4.06 in the future you can always come back and modify the constraints in opam-repository by opening a PR here.